samples/default-field-value/README.md

```diff
@@ -0,0 +1,50 @@
+## Default field values
+
+This UI Extension uses configuration parameters to set a default value for a field.
```

Comment:
You don't say what this thing is and start with parameters. I'd prefer something like: "This UI Extension is a text editor for which a default field value can be defined."

Reply:
done